fearless/Modules/Staking/StakingMain/StakingMainViewController.swift

```diff
@@ -55,7 +74,31 @@ final class StakingMainViewController: UIViewController, AdaptiveDesignable {
         presenter.performAccountAction()
     }
 
+    @IBAction func toggleNetworkWidgetVisibility(sender: ActionTitleControl) {
+        amountInputView.textField.resignFirstResponder()
+        let animationOptions: UIView.AnimationOptions = sender.isActivated ? .curveEaseIn : .curveEaseOut
+
+        self.storiesViewZeroHeightConstraint?.isActive = sender.isActivated
+        UIView.animate(
+            withDuration: 0.35,
+            delay: 0.0,
+            options: [animationOptions],
+            animations: {
+                self.view.layoutIfNeeded()
```

Comment:
Why do we need to call this twice?

Reply:
You are right, the second one is not necessary